html/inc/account.inc

```diff
@@ -16,7 +16,18 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with BOINC.  If not, see <http://www.gnu.org/licenses/>.
 
-// forms for account creation and login
+// functions related to account creation and login:
+// - forms for create / login
+// - function to make login token
+
+// make login token, store in user record, return token
+//
+function make_login_token($user) {
+    $token = substr(random_string(), 0, 8);
```

Comment:
this should be login_token (or the sql below should reference the token variable)